src/main/java/com/amihaiemil/docker/SystemDiskSpaceInfo.java

```diff
@@ -0,0 +1,60 @@
+package com.amihaiemil.docker;
+
+import javax.json.JsonArray;
+import javax.json.JsonObject;
+
+/**
+ * Docker disk space usage information.
+ * @author Boris Kuzmic (boris.kuzmic@gmail.com)
+ * @since 0.0.6
+ */
+public final class SystemDiskSpaceInfo implements DiskSpaceInfo {
+
+    /**
+     * Response Json object from system/df call.
+     */
+    private final JsonObject json;
+
+    /**
+     * Ctor.
+     * @param jsonObject Response Json from system/df call
+     */
+    SystemDiskSpaceInfo(final JsonObject jsonObject) {
+        this.json = jsonObject;
+    }
+
+    @Override
+    public Long images() {
+        return this.json.getJsonNumber("LayersSize").longValue();
+    }
+
+    @Override
+    public Long containers() {
+        Long totalContainersSpace = 0L;
+        JsonArray containers = this.json.getJsonArray("Containers");
+        for (int i=0; i<containers.size(); i++) {
+            JsonObject container = containers.getJsonObject(i);
+            totalContainersSpace +=
+                    container.getJsonNumber("SizeRootFs").longValue();
+        }
```

Comment:
@bkuzmic Insted of these ``for``s, is it possible to use some Java 8 features like ``.forEach()`` or Stream API?

Reply:
@amihaiemil I've changed it to Stream API. It's shorter, but I don't really like the casting to JsonObject. But if you are ok with it, we can use streams like that.